storage/mysql/admin_storage_test.go

```diff
@@ -241,12 +241,17 @@ func TestCheckDatabaseAccessible_OK(t *testing.T) {
 	}
 }
 
-func setNulls(ctx context.Context, db *sql.DB, treeID int64) error {
+func setNulls(ctx context.Context, t *testing.T, db *sql.DB, treeID int64) error {
```

Comment:
nit: Since you're making this a test helper, could simply it further by removing the `error` return value and just calling `t.Fatal()` instead.